methods/lcarun.ts

```diff
@@ -2,28 +2,102 @@ import knex from 'knex';
 
 import { Lci } from 'models/schema';
 import { RunParams } from './lca.model';
-// import { listenerCount } from 'cluster';
+
+// NonrenewableUnitParams were used to change units for nonrenewable group. Units change is not needed for other groups
+interface NonrenewableUnitParams {
+  'brownCoal': number;
+  'hardCoal': number;
+  'crudeOil': number;
+  'mineGas': number;
+  'naturalGas': number;
+  'Uranium': number;
+  [key: string]: number;
```

Comment:
What is this used for?

Reply:
add comments for this part in the new branch, KL/lci_calculation